Clinical-Genomics/scout
# Update individual docs + tiny bugfix

This PR:
- Adds documentation describing the s`scout update individual` command (fix #2527)
- Fixes a minor bug happening when running the update individual command by specifying right case id but wrong individual name.

**How to test**:
1. Documentation should be easy to read
1. To test the bug fix, use a local instance and run this command both on master and this branch: 
```
scout --demo update individual -c internal_id -n hello
```

**Expected outcome**:
When running the command above from this branch there should be no crash

**Review:**
